stock_move_pivot_total_price/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Bernat Obrador
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    product_total_price = fields.Float(
```

Comment:
This module is going to need a pre-init hook, it will be almost uninstallable on large databases. The pre-init hook will need to firstly create the column, and then populate it. Population will be less than trivial having to take account of list_price, and then the sum of extra prices from attribute values.